mdtraj/geometry/order.py

```diff
@@ -0,0 +1,263 @@
+##############################################################################
+# MDTraj: A Python Library for Loading, Saving, and Manipulating
+#         Molecular Dynamics Trajectories.
+# Copyright 2012-2015 Stanford University and the Authors
+#
+# Authors: Christoph Klein
+# Contributors:
+#
+# MDTraj is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with MDTraj. If not, see <http://www.gnu.org/licenses/>.
+##############################################################################
+
+from __future__ import print_function, division
+
+import numpy as np
+NP18 = float(np.__version__[:3]) >= 1.8
+
+from mdtraj.geometry.distance import compute_center_of_mass
+from mdtraj.utils import ensure_type
+from mdtraj.utils.six import string_types
+
+
+__all__ = ['compute_nematic_order', 'compute_inertia_tensor']
+
+
+def compute_nematic_order(traj, indices='chains'):
+    """Compute the nematic ordering of a group in every frame.
+
+    Parameters
+    ----------
+    traj : Trajectory
+        Trajectory to compute ordering in.
+    indices : str or list of lists, optional, default='chains'
+        The group to consider. Users can pass their own indices as a list of
+        lists with the "shape" (n_compounds, len(each_compound)).
+        Recognized string keywords are 'chains' and 'residues'.
+
+    Returns
+    -------
+    S2 : np.ndarray, shape=(traj.n_frames,), dtype=float64
+        Nematic order parameter values in every frame.
+
+    References
+    ----------
+    .. [1] Allen, M. P.; Tildesley , D. J. (1987), "Computer Simulation of
+           Liquids", Ch. 11.5
+    .. [2] http://cmt.dur.ac.uk/sjc/thesis_dlc/node65.html
+    .. [3] http://cmt.dur.ac.uk/sjc/thesis_dlc/node19.html
+
+    Examples
+    --------
+    Ordering of chains in an alkylsilane monolayer of C10H31-Si(OH)2-:
+
+    >>> import mdtraj as md
+    >>> from mdtraj.testing import get_fn
+    >>> traj = md.load(get_fn('monolayer.xtc'), top=get_fn('monolayer.pdb'))
+    >>> # Each of the 100 chains contains 36 atoms.
+    >>> chain_indices = [[n+x for x in range(36)] for n in range(0, 3600, 36)]
+    >>> S2 = md.compute_nematic_order(traj, indices=chain_indices)
+
+    The chains were attached to a flat, crystalline substrate and are thus
+    highly ordered with a mean S2 of ~0.996.
```

Comment:
Is the value always between 0 and 1. If so, maybe that can go in the short description https://github.com/mdtraj/mdtraj/pull/759/files#r26452883